MantidQt/CustomInterfaces/src/Reflectometry/QtReflEventView.cpp

```diff
@@ -34,12 +48,49 @@ IReflEventPresenter *QtReflEventView::getPresenter() const {
   return m_presenter.get();
 }
 
-/** Returns the number of time slices
-* @return :: The number of time slices
+/** Returns the time slicing value(s) obtained from the selected widget
+* @return :: Time slicing values
 */
-std::string QtReflEventView::getTimeSlices() const {
+std::string QtReflEventView::getTimeSlicingValues() const {
+
+  std::string values;
+
+  if (m_sliceType == m_slicingTypes[0])
+    values = m_ui.uniformEvenEdit->text().toStdString();
+  else if (m_sliceType == m_slicingTypes[1])
+    values = m_ui.uniformEdit->text().toStdString();
+  else if (m_sliceType == m_slicingTypes[2])
+    values = m_ui.customEdit->text().toStdString();
+  else if (m_sliceType == m_slicingTypes[3])
+    values = m_ui.logValueComboBox->currentText().toStdString();
```

Comment:
Just a very minor point: I think this would be easier to read if we replaced `m_slicingTypes` with the actual values we are using for the comparisons. I had to go back to `QtReflEventView.h` to check how the vector was initialized. Is there any reason not to do this?